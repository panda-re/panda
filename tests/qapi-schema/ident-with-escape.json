# we allow escape sequences in strings, if they map back to ASCII
# { 'command': 'fooA', 'data': { 'bar1': 'str' } }
{ 'c\u006fmmand': '\u0066\u006f\u006FA',
  'd\u0061ta': { '\u0062\u0061\u00721': '\u0073\u0074\u0072' } }
