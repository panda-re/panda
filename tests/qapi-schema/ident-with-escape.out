enum QType ['none', 'qnull', 'qint', 'qstring', 'qdict', 'qlist', 'qfloat', 'qbool']
    prefix QTYPE
command fooA q_obj_fooA-arg -> None
   gen=True success_response=True boxed=False
object q_empty
object q_obj_fooA-arg
    member bar1: str optional=False
